### PR TITLE
7.0.0 release changes

### PR DESCRIPTION
**Pull request by nsharma-git**
# Description

Update a few test validation steps

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Local docker environment

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-debian10/topic_data_creation.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@
       shell: |
         kafka-topics --create --topic test-replicator-source \
           --bootstrap-server mds-kafka-broker1:9091 \
-          --command-config /etc/kafka/client.properties
+          --command-config /etc/kafka/client.properties \
+          --replication-factor 1 --partitions 1
       run_once: true
       register: output
       failed_when:
```
Comment (domenicbove) [inline molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-debian10/topic_data_creation.yml:14]: NIT: but this file is just a duplicate of each other, could move it down a directory and just use relative path to it

File: roles/kafka_broker/tasks/rbac_rolebindings.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@
 # confluent iam rolebinding create --principal User:<audit-log-admin> --role ResourceOwner --resource Topic:confluent-audit-log-events --prefix --cluster-name audit_logs
 - name: Grant Audit Logs Principal ResourceOwner on confluent-audit-log-events Prefixed Topics
   uri:
-    url: "http://mds-kafka-broker1:8090/security/1.0/principals/User:{{audit_logs_destination_principal}}/roles/ResourceOwner/bindings"
+    url: "{{mds_bootstrap_server_urls.split(',')[0]}}/security/1.0/principals/User:{{audit_logs_destination_principal}}/roles/ResourceOwner/bindings"
     method: POST
```
Comment (domenicbove) [inline roles/kafka_broker/tasks/rbac_rolebindings.yml:35]: task below needs this as well

**Review by domenicbove — CHANGES_REQUESTED** [review]
Looks great, one needed change!

File: roles/kafka_broker/tasks/rbac_rolebindings.yml (hunk 2 above)
Comment (nsharma-git) [reply roles/kafka_broker/tasks/rbac_rolebindings.yml:35]: Done

File: molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-debian10/topic_data_creation.yml (hunk 1 above)
Comment (nsharma-git) [reply molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-debian10/topic_data_creation.yml]: Done

File: molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml [hunk 3]
```diff
@@ -12,4 +12,4 @@
   import_playbook: confluent.platform.all
 
 - name: Create Topics and Data for Replication on Source Cluster
-  import_playbook: topic_data_creation.yml
```
Comment (domenicbove) [inline molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml:15]: looks like this file was deleted

**Comment by domenicbove** [comment]
Need to add this to `molecule/Dockerfile-debian-archive.j2`
```
# Hack to fix cacerts on debian hosts, preventing confluenthub installs from working
RUN /usr/bin/printf '\xfe\xed\xfe\xed\x00\x00\x00\x02\x00\x00\x00\x00\xe2\x68\x6e\x45\xfb\x43\xdf\xa4\xd9\x92\xdd\x41\xce\xb6\xb2\x1c\x63\x30\xd7\x92' > /etc/ssl/certs/java/cacerts && \
    /var/lib/dpkg/info/ca-certificates-java.postinst configure
```

File: molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml (hunk 3 above)
Comment (nsharma-git) [reply molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml:15]: We moved the file a level up and removed form each scenario directories

Comment (domenicbove) [reply molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml:15]: yes this PR does not show that the file was moved, just deleted

Comment (domenicbove) [reply molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml:15]: NVRM i see it now, thanks!

File: molecule/topic_data_creation.yml [hunk 4]
```diff
@@ -17,5 +18,5 @@
     - name: Create Topic Data
       shell: |
         seq 10 | kafka-console-producer --topic test-replicator-source \
-         --broker-list mds-kafka-broker1:9091 --producer.config /etc/kafka/client.properties
+         --broker-list mds-kafka-broker1:9092 --producer.config /etc/kafka/client.properties
```
Comment (domenicbove) [inline molecule/topic_data_creation.yml:21]: dont change this line, the client.properties file is written to match a specific listener

Comment (nsharma-git) [reply molecule/topic_data_creation.yml]: Done

**Review by domenicbove — APPROVED** [review]
Nice LGTM!

**Review by domenicbove — APPROVED** [review]
LGTM!

**Pull request merged**